.github/config.yml

```diff
@@ -0,0 +1,34 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉for opening your very first issue in this project. AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/portal/blob/develop/code_of_conduct.md).🙌
+  You may submit a PR if you like, make sure to follow our [Pull Request Template](https://github.com/anitab-org/portal/blob/develop/PULL_REQUEST_TEMPLATE.md)! If you want to report a bug🐞 please follow our [Issue Template](https://github.com/anitab-org/portal/blob/develop/ISSUE_TEMPLATE.md). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
+  
+  
+  Feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).💖 We have different streams for each active repository for discussions.✨ Hope you have a great time there!😄
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉 for opening your first pull request.✨ AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸Please adhere to our [Code of Conduct](https://github.com/anitab-org/portal/blob/develop/code_of_conduct.md) and [Contribution Guidelines](https://github.com/anitab-org/portal/blob/develop/CONTRIBUTING.md).🙌.We will get back to you as soon as we can.😄
```

Comment:
Here there is a full stop before and after the emoji. Can you remove it from one place.